lib/model/query/entities.js

```diff
@@ -158,13 +168,13 @@ const createVersion = (dataset, entity, data, label, userAgentIn = null) => ({ c
 
   // TODO: revisit sources.
   // For now, attach a dummy 'api' entity def source to return for PATCH request.
-  const source = new Entity.Def.Source({
+  const source = new Entity.Def.SourceSummary({
     type: 'api',
     details: null
   });
 
   return one(sql`
-  with def as (${_defInsert(entity.id, null, creatorId, userAgent, label, json)}),
+  with def as (${_defInsert(entity.id, creatorId, userAgent, label, json)}),
```

Comment:
shouldn't we create a row in source table when entity is created/updated via API?